app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -38,7 +38,7 @@ class RouteDisplayState(
  * A clickable bar with Start Location name
  * A clickable bar with Destination name
  */
-class RouteOptionState(
+class RouteOptionStateTempName(
     startLocation: Location,
```

Comment:
Theere was an overlapping name here with my SearchState, so I just changed the name temporarily

Reply:
wait what does this mean

Reply:
One the classes in SearchState had the same name as a class in RouteCard State. This created an error and wouldn't let me compile my code